extensions/interactions/AlgebraicExpressionInput/directives/oppia-interactive-algebraic-expression-input.component.ts

```diff
@@ -66,7 +66,7 @@ export class AlgebraicExpressionInputInteractionComponent
     private translateService: TranslateService,
   ) {}
 
-  isCurrentAnswerValid(checkForTouched = true): boolean {
+  isCurrentAnswerValid = (checkForTouched = true): boolean => {
```

Comment:
Why did this change? Please undo, we don't use this syntax in the rest of the file.

Ditto below where function definition syntax has changed.

Reply:
Because this function is being passed as a callback [here](https://github.com/oppia/oppia/pull/18379/files/904fed8ff6613690e942509a0ef01757cd4e23ec#diff-b7cb87d97ce2e55b62878600b671096181cd31b6deb588cfd18ba6ee8a576286R140). Converted it to an arrow function so that `this` continues to refer to the lexical scope object even when called in `this.currentInteractionService.registerCurrentInteraction`

Reply:
Would it make sense to use bind in that case and use the same syntax as earlier? i.e. where the callback is passed, bind the scope.

Reply:
I have a preference for arrow functions since they are cleaner and easier to read. These files also have many other callback functions like `submitAnswer` , `onAnswerChange` that have also been written as arrow functions so I think it should be okay

Reply:
I'm not entirely convinced about deviating from the existing pattern for the reasons mentioned (i.e. cleaner, readable) -- the current pattern used seems to be clean / readable too. Other devs might be confused about why arrow functions are used in some places vs. the existing pattern. 

Also, we currently use the bind method in the item selection input interaction:

https://github.com/oppia/oppia/blob/441a01437ee86cd91f95d5d36cfd2a232fd2a8b3/extensions/interactions/ItemSelectionInput/directives/oppia-interactive-item-selection-input.component.ts#L135-L136

and the conversation skin component

https://github.com/oppia/oppia/blob/441a01437ee86cd91f95d5d36cfd2a232fd2a8b3/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts#L442-L443

So I'd recommend using that instead. Thanks!

Reply:
done everywhere